addons/l10n_vn/migrations/17.0.2.0.3/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,7 @@
+from odoo import api, SUPERUSER_ID
```

Comment:
Thanks for the migration script! In local migrations scripts, you do not need to specify odoo version, you can name this folder `2.0.3`

Reply:
Done